### PR TITLE
rename "test/helpers" to "test/apiCallers"

### PR DESCRIPTION
**Pull request by seanim0920**

### CODE REVIEW

File: APILambda/test/userFlows/events.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { CreateEventInput } from "../../events/createEvent.js"
+import { callCreateEvent, callJoinEvent } from "../apiCallers/events.js"
+import { testEvent } from "../testEvents.js"
+import { createUserFlow } from "./users.js"
+
+export const createEventFlow = async (eventInput: Partial<CreateEventInput>[]): Promise<{attendeeToken: string, hostToken: string, eventIds: number[]}> => {
```
Comment (seanim0920) [inline APILambda/test/userFlows/events.ts:6]: Add example as a comment

Comment (seanim0920) [inline APILambda/test/userFlows/events.ts:6]: "createEventWithAttendeeFlow" & "createEventFlow"

File: APILambda/test/userFlows/users.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+export const createUserFlow = async (): Promise<{handle: string, userId: string, token: string, name: string}> => {
+  if (testUserCounter.currentUserIndex >= global.users.length) {
+    throw new Error("used all test users")
+  }
+
+  const testUser = global.users[testUserCounter.currentUserIndex]
+  const { body: { id: userId, handle } } = await callPostUser(testUser.auth)
+  const token = await testUser.refreshAuth()
+
+  testUserCounter.currentUserIndex = (testUserCounter.currentUserIndex + 1)
```
Comment (seanim0920) [inline APILambda/test/userFlows/users.ts:15]: testUserCounter.currentUserIndex++

**Comment by seanim0920** [comment]
https://github.com/tifapp/FitnessProjectBackend/actions/runs/7319871367/job/19938242059?pr=134
https://github.com/tifapp/FitnessProjectBackend/pull/135

**Review by suryatiru — APPROVED** [review]

**Pull request merged**